x-pack/elastic-agent/magefile.go

```diff
@@ -595,7 +595,7 @@ func packageAgent(requiredPackages []string, packagingFn func()) {
 		defer os.RemoveAll(dropPath)
 		defer os.Unsetenv(agentDropPath)
 
-		packedBeats := []string{"filebeat", "heartbeat", "metricbeat"}
+		packedBeats := []string{"filebeat", "heartbeat", "metricbeat", "osquerybeat"}
```

Comment:
I'm curious if we checked how the image size changes with this PR.

Reply:
osquerybeat is fairly small, it's 35 mb or so.